contracts/cw20/merkleAirdrop/contract.ts

```diff
@@ -316,5 +398,99 @@ export const CW20MerkleAirdrop = (
     }
   }
 
-  return { instantiate, use }
+  const messages = () => {
+    const instantiate = (
+      codeId: number,
+      label: string,
+      msg: Record<string, unknown>
+    ): InstantiateMessage => {
+      return {
+        txSigner,
+        codeId: codeId,
+        label: label,
+        msg: msg,
+        funds: [],
+        admin: txSigner,
+      }
+    }
+
+    const registerAndReleaseEscrow = (
+      airdropAddress: string,
+      merkleRoot: string,
+      start: Expiration,
+      expiration: Expiration,
+      // totalAmount: number,
+      stage: number
+    ): [RegisterMessage, ReleaseEscrowMessage] => {
+      return [
+        {
+          sender: txSigner,
+          contract: airdropAddress,
+          msg: {
+            register_merkle_root: {
+              merkle_root: merkleRoot,
+              start,
+              expiration,
+            },
+          },
+          funds: [],
+        },
+        {
+          sender: txSigner,
+          contract: ESCROW_CONTRACT_ADDRESS,
+          msg: {
+            release_locked_funds: {
+              airdrop_addr: airdropAddress,
+              stage,
+            },
+          },
+          funds: [],
+        },
+      ]
+    }
+
+    const depositEscrow = (airdropAddress: string): DepositEscrowMessage => {
+      return {
+        sender: txSigner,
+        contract: ESCROW_CONTRACT_ADDRESS,
+        msg: {
+          lock_funds: {
+            airdrop_addr: airdropAddress,
+          },
+        },
+        funds: [
+          coin(ESCROW_AMOUNT * 1000000, getNetworkConfig(NETWORK).feeToken),
+        ],
+      }
+    }
+
+    const claim = (
+      airdropAddress: string,
+      stage: number,
+      amount: string,
+      proof: string[]
+    ): ClaimMessage => {
+      return {
+        sender: txSigner,
+        contract: airdropAddress,
+        msg: {
+          claim: {
+            stage,
+            amount,
+            proof,
+          },
+        },
+        funds: [],
+      }
+    }
+
+    return {
+      instantiate,
+      registerAndReleaseEscrow,
+      depositEscrow,
+      claim,
+    }
+  }
+
+  return { instantiate, use, messages }
```

Comment:
This file is huge. I like to keep files under 200 lines. Is it worth the time to break it up a bit?

Reply:
For now I think we can leave it like this but maybe come back in the future to split it